pkg/destroy/powervs/cloudobjectstorage.go

```diff
@@ -0,0 +1,226 @@
+package powervs
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/resourcecontrollerv2"
+	"github.com/pkg/errors"
+)
+
+const cosTypeName = "cos instance"
+const cosResourceID = "dff97f5c-bc5e-4455-b470-411c3edbe49c"
+
+// listCOSInstances lists COS service instances.
+func (o *ClusterUninstaller) listCOSInstances() (cloudResources, error) {
```

Comment:
can this code be made common for ibmcloud/powervs in the future?

Reply:
i think we'll end up removing this when we get the bootstrap phase in anyway, b/c it should delete this. the only reason we use it is for the bootstrap's ignition config (cc @miyamotoh )

Reply:
This does not hurt anything by leaving it in.  And if @miyamotoh code does not clean up the ignition file for whatever reason, the current code will still delete it.